user-guide.md

```diff
@@ -82,9 +151,11 @@ source env/bin/activate
 .\env\Scripts\activate
 ```
 ​
-## Installing the encryption tool
+### Installing the crypt4gh encryption tool
 ​
-The next step is to install [crypt4gh](https://github.com/EGA-archive/crypt4gh), by running the following command:
+The next step is to install
+[crypt4gh](https://github.com/EGA-archive/crypt4gh), by running the
+following command:
```

Comment:
Another option if a static binary is preferred, it to use the GO version available from here: https://github.com/elixir-oslo/crypt4gh/releases/latest